spec/presenters/two_factor_authentication/sign_in_webauthn_platform_selection_presenter_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe TwoFactorAuthentication::SignInWebauthnPlatformSelectionPresenter do
+  let(:user) { create(:user) }
+  let(:configuration) { create(:webauthn_configuration, user: user) }
+
+  let(:presenter) do
+    described_class.new(user: user, configuration: configuration)
+  end
+
+  describe '#type' do
+    it 'returns webauthn_platform' do
+      expect(presenter.type).to eq 'webauthn_platform'
+    end
+  end
+
+  describe '#label' do
+    it 'raises with missing translation' do
+      expect(presenter.label).to eq(
+        t('two_factor_authentication.two_factor_choice_options.webauthn_platform'),
+      )
+    end
+  end
+
+  describe '#info' do
+    it 'raises with missing translation' do
```

Comment:
I just noticed these as I start work on a related ticket. These methods don't raise, so I think we should revise the expectation description accordingly.

```suggestion
    it 'returns the label text' do
      expect(presenter.label).to eq(
        t('two_factor_authentication.two_factor_choice_options.webauthn_platform'),
      )
    end
  end

  describe '#info' do
    it 'returns the info text' do
```